packages/token-bridge-sdk/src/util/subgraph.ts

```diff
@@ -42,11 +82,14 @@ type FetchL2BlockNumberFromSubgraphQueryResult = {
   }
 }
 
-export async function fetchL2BlockNumberFromSubgraph(
+export const fetchBlockNumberFromSubgraph = async (
+  chainType: 'L1' | 'L2',
   l2ChainId: number
-): Promise<number> {
-  const queryResult: FetchL2BlockNumberFromSubgraphQueryResult =
-    await getL2SubgraphClient(l2ChainId).query({
+): Promise<number> => {
+  const queryResult: FetchBlockNumberFromSubgraphQueryResult =
+    await (chainType === 'L2' ? getL2SubgraphClient : getL1SubgraphClient)(
+      l2ChainId
+    ).query({
```

Comment:
Let's split these up into two statements to make it more readable:

```suggestion
  const client = chainType === 'L2' ? getL2SubgraphClient(l2ChainId) : getL1SubgraphClient(l2ChainId)
    
  const queryResult: FetchBlockNumberFromSubgraphQueryResult = await client.query({
```

Reply:
Done